src/test/javascript/spec/component/text-editor/text-editor.component.spec.ts

```diff
@@ -71,16 +77,7 @@ describe('TextEditorComponent', () => {
 
     beforeEach(async () => {
```

Comment:
did you forget to remove the async here?

Reply:
```suggestion
    beforeEach(() => {
```